enterprise/api/v1/views.py

```diff
@@ -1691,3 +1694,44 @@ def link_user(self, request, pk=None):
             enterprise_user.save()
 
         return Response(response_body, status=HTTP_200_OK, headers=headers)
+
+
+class PlotlyAuthView(generics.GenericAPIView):
+    """
+    API to authenticate user with Plotly.
+    """
+    permission_classes = (IsAuthenticated,)
+
+    @permission_required(
+        'enterprise.can_access_admin_dashboard',
+        fn=lambda request, enterprise_uuid: enterprise_uuid
+    )
+    def get(self, request, enterprise_uuid):
+        """
+        Generate auth token for plotly.
+        """
+        # TODO: Update endpoint. The logic in this endpoint is temporarily needed for a POC.
+
+        # This is a new secret key and will be only shared between LMS and our Plotly server.
+        secret_key = settings.ENTERPRISE_PLOTLY_SECRET
+
+        now = int(time())
+        expires_in = 3600 # time in seconds after which token will be expired
+        exp = now + expires_in
+
+        AUDIENCE = 'dash.tests:8050' # TBD: Should be a appropriate value?
```

Comment:
we can set this to enterprise-plotly.edx.org ... or not at all since AUDIENCE claim in optional